src/core/providers/strategies/helpers/getPendingTransactionsHandlers.ts

```diff
@@ -0,0 +1,25 @@
+import { PendingTransactionsStateManager } from 'core/managers/internal/PendingTransactionsStateManager/PendingTransactionsStateManager';
+import { ProviderErrorsEnum } from 'types/provider.types';
+
+export async function getPendingTransactionsHandlers<T>(props?: {
```

Comment:
this function --> can it be used in the classes ?

Reply:
No, it is different